include/class.thread.php

```diff
@@ -1319,12 +1319,14 @@ function saveEmailInfo($vars) {
 
     /* static */
     function logEmailHeaders($id, $mid, $header=false) {
+        $headerInfo = Mail_Parse::splitHeaders($header);
```

Comment:
You probably should parse header when ready to use.  We can potentially return without using it.